packages/xstate-react/test/typegenTypes.test.typegen.ts

```diff
@@ -0,0 +1,301 @@
+// This file was automatically generated. Edits will be overwritten
```

Comment:
this file shouldn't be committed - we probably need some option in the extension to disallow typegen for the whole workspace